.github/workflows/ci.yml

```diff
@@ -137,6 +137,7 @@ jobs:
           - host: windows-latest
             target: aarch64-pc-windows-msvc
             # target not supported by the 'ring' rust dependency (ssl library partially built by gcc)
+            # https://github.com/briansmith/ring/issues/1167
```

Comment:
I wonder if we should switch to a different crypto implementation...

This was also the blocker when trying to build for other "exotic" targets like wasm

Reply:
`ring` is a constant struggle in all my build targeting, would be really nice to have a pure-rust implementation, but I don't know if one exists.

Friend of a friend has a fun post on the lengths he went to to build `ring` for his custom phone product: https://www.bunniestudios.com/blog/?p=6521